force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +121,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String result = '';
+        if(a==Null){
+            result = Null;
+        }else if(math.mod(a, 2)==0){
+            result = 'Even';
+        }else{
+            result = 'Odd';
+        }
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Good job.  Just as an example of another solution I like to solve these sort of problems with a ternary operator. 

```
  public static String checkEvenOdd(Integer a) {
        if(a == null ){
            return null;
        }
        return Math.mod(a, 2) == 0 ? 'Even' : 'Odd';
    }
 ```